src/main/groovy/wooga/gradle/paket/unity/PaketUnityPlugin.groovy

```diff
@@ -56,44 +57,64 @@ class PaketUnityPlugin implements Plugin<Project> {
+
+                if (!extension.paketReferencesFiles || extension.paketReferencesFiles.empty) {
+                    logger.warn("Will skip ${it.name} as there are no paket reference files were found in the project")
+                    return false
+                }
+
+                if (!extension.paketDependenciesFile || !extension.paketDependenciesFile.exists()) {
+                    logger.warn("Will skip ${it.name} as there are no paket dependencies files were found in the project")
+                    return false
+                }
+
+                if (!extension.paketLockFile || !extension.paketLockFile.exists()){
+                    logger.warn("Will skip ${it.name} as there are no paket lock files were found in the project")
+                    return false
+                }
+
+                return true
             }
         }
 
         configurePaketDependencyInstallIfPresent()
     }
 
+    private static void configureExtension(DefaultPaketUnityPluginExtension extension, Project project) {
+        extension.paketUpmPackageEnabled.convention(PaketUnityPluginConventions.paketUpmPackageEnabled.getBooleanValueProvider(project))
+        extension.defaultNamespace.convention(PaketUnityPluginConventions.defaultNamespace.getStringValueProvider(project))
+    }
+
     private static void createPaketUnityInstallTasks(final Project project, final PaketUnityPluginExtension extension) {
-        def installProviders = extension.paketReferencesFiles.files.collect { referenceFile ->
-            def taskName = INSTALL_TASK_NAME + referenceFile.parentFile.name
+
+        // Create an install task for EACH paket.unity3d.references file
+        def installProviders = extension.paketReferencesFiles.files.collect { referencesFile ->
+            def taskName = INSTALL_TASK_NAME + referencesFile.parentFile.name
             def installProvider = project.tasks.register(taskName, PaketUnityInstall)
             installProvider.configure { PaketUnityInstall t ->
                 t.group = GROUP
-                t.description = "Installs dependencies for Unity3d project ${referenceFile.parentFile.name} "
-                t.conventionMapping.map("paketOutputDirectoryName", { extension.getPaketOutputDirectoryName() })
+                t.description = "Installs dependencies for Unity3d project ${referencesFile.parentFile.name} "
                 t.conventionMapping.map("includeAssemblyDefinitions", { extension.getIncludeAssemblyDefinitions() })
-                t.conventionMapping.map("assemblyDefinitionFileStrategy", { extension.getAssemblyDefinitionFileStrategy() })
                 t.conventionMapping.map("preInstalledUpmPackages", { extension.getPreInstalledUpmPackages() })
                 t.frameworks = extension.getPaketDependencies().getFrameworks()
                 t.lockFile = extension.getPaketLockFile()
-                t.referencesFile = referenceFile
+                t.defaultNamespace.convention(extension.defaultNamespace)
```

Comment:
Rename this to something that makes more clear that this is about the generated UPM packages, something like `defaultUPMNamespace`

Reply:
Will do.